moziris/email_report/email_client.py

```diff
@@ -51,22 +50,18 @@ def get_file_attachment():
                                   filename=os.path.basename(test_report_file))
             return attachment
         else:
-            raise Exception('File {} is not present in path'.format(test_report_file))
+            raise Exception('File %s is not present in path' % test_report_file)
 
     def send_email_report(self, target: str, test_status: str, repo_details: str):
         email = MIMEMultipart()
         body_message = ""
         if isinstance(repo_details, dict):
-            body_message = MIMEText((
-                ' Repo_details:\n'
-                ' Branch_name: {}\n'
-                ' Branch_head: {}\n'
-                ' Test_Run_Details: {}\n'
-                ' Note: To see the complete run output, please check the attachment.'.format(
-                    repo_details.get('iris_branch'), 
-                    repo_details.get('iris_branch_head'), 
-                    test_status)
-                )) 
+            body_message = MIMEText(
+                ''' Repo_details:\n ''' + """Branch_name:""" + repo_details.get(
+                    'iris_branch')
+                + " \n " + '''Branch_head: ''' + repo_details.get(
+                    'iris_branch_head') + "\n\n" + '''Test_Run_Details: ''' + test_status +
+                ''' \nNote: To see the complete run output, please check the attachment.''')
         else:
```

Comment:
What's reason for the change in string interpolation? The original code is using the new implementation for python 3

Reply:
Good catch. My code went out of sync with the master repo during the rewrite, and old code crept back in.